src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -187,7 +187,8 @@ std::unique_ptr<model_spec> drawing_classifier::init_model() const {
       /* input               */ input_name,
       /* num_output_channels */ 128,
       /* num_input_channels  */ 64 * 3 * 3,
-      /* weight_init_fn      */ initializer);
+      /* weight_init_fn      */ initializer,
+      /* bias_init_fn        */ zero_weight_initializer());
```

Comment:
Nice catch! In protobuf it's implicitly zero-initialized. But explicit is better!